src/vos/vos_dtx.c

```diff
@@ -2225,7 +2225,7 @@ vos_dtx_set_flags(daos_handle_t coh, struct dtx_id *dti, uint32_t flags)
 	if (rc == -DER_NONEXIST) {
 		rc = dbtree_lookup(cont->vc_dtx_committed_hdl, &kiov, &riov);
 		if (rc == 0) {
-			D_ERROR("Not allow to set flag on committed/aborted DTX entry "DF_DTI"\n",
+			D_ERROR("Not allow to set flag on committed (1) DTX entry "DF_DTI"\n",
```

Comment:
minor nit s/allow/allowed

Reply:
The line# may be changed for different branches if we add more patches in the future, but the (1) and (2) can be easily located.

Reply:
Sometime, the user may just reports the error message but not exactly to know the system version, then with (1) and (2) message, we can easily to know at which point we hit the failure.